### PR TITLE
fix(storage): change HTTP methode for the route to get s3UserCredentials from GET to POST

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`  
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-114046 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Related

- [x] TRANS-52522

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js [hunk 1]
```diff
@@ -38,7 +38,17 @@ export default class PciStoragesColdArchiveService {
   getS3Credentials(projectId, userId) {
     return this.$http
       .get(`/cloud/project/${projectId}/user/${userId}/s3Credentials`)
-      .then(({ data }) => data);
+      .then(({ data }) => {
+        return data?.[0];
+      });
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js:43]:
```suggestion
      .then(({ data }) => data?.[0]);
```

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js [hunk 2]
```diff
@@ -29,7 +29,17 @@ export default class PciStoragesObjectStorageService {
...
+        return data?.[0];
+      });
+  }
+
+  getS3Secret(projectId, userId) {
+    return this.$http
+      .post(`/cloud/project/${projectId}/user/${userId}/s3Credentials`)
+      .then(({ data }) => {
+        return data;
+      });
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js:42]:
```suggestion
      .then(({ data }) => data);
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js (hunk 1 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js]: Fixed

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js [hunk 3]
```diff
@@ -29,7 +29,17 @@ export default class PciStoragesObjectStorageService {
   getS3Credentials(projectId, userId) {
     return this.$http
       .get(`/cloud/project/${projectId}/user/${userId}/s3Credentials`)
-      .then(({ data }) => data);
+      .then(({ data }) => {
+        return data?.[0];
+      });
```
[the remark this answers is not in the document]
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js]: Fixed

File: packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js (hunk 2 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.service.js]: Fixed

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js [hunk 4]
```diff
@@ -38,7 +38,19 @@ export default class PciStoragesColdArchiveService {
...
+  }
+
+  getS3Secret(projectId, userId, userAccess) {
+    return this.$http
+      .post(
+        `/cloud/project/${projectId}/user/${userId}/s3Credentials/${userAccess}/secret`,
+      )
+      .then(({ data }) => {
+        return data;
+      });
```
[the remark this answers is not in the document]
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js]: Fixed

**Review by anooparveti — APPROVED** [review]

**Comment by anooparveti** [comment]
Pulling the translations

**Review by anooparveti — APPROVED** [review]

**Pull request merged**